presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/TransformCorrelatedInPredicateToJoin.java

```diff
@@ -335,7 +335,7 @@ public DecorrelatingVisitor(Lookup lookup, Iterable<Symbol> correlation)
                 return new Decorrelated(
                         decorrelated.getCorrelatedPredicates(),
                         new ProjectNode(
-                                node.getId(), // FIXME should I reuse or not?
+                                node.getId(),
```

Comment:
just a thought: In general I am not sure I know why we tend to reuse node ids. What benefits does it give? It is not always possible, so more coherent would be to use new id always.

CC: @sopel39?